unstructured/ingest/v2/processes/connectors/onedrive.py

```diff
@@ -123,12 +123,12 @@ def drive_item_to_file_data(self, drive_item: "DriveItem") -> FileData:
         server_path = file_path + "/" + filename
         rel_path = server_path.replace(self.index_config.path, "").lstrip("/")
         date_modified_dt = (
-            parser.parse(drive_item.last_modified_datetime)
+            parser.parse(str(drive_item.last_modified_datetime))
```

Comment:
This was raising a `json.decoder.JSONDecodeError` when trying to use `parser.parse(drive_item.last_modified_datetime)`